database/scripts/development-database/create_dev_schema.sql

```diff
@@ -59,7 +59,7 @@ SELECT
 FROM
     public.regions
 WHERE
-    ogc_fid IN (1, 2, 3, 4, 8, 9, 14, 28, 31);
+    ogc_fid IN (3, 11);
```

Comment:
there are only two regions that we use for testing? maybe just add a comment here, which regions these refer to.

Reply:
Yes only two regions are used. Heidelberg and a region in Algeria. In addition to those GeoJSON fixtures are sometimes used. I added a comment to which regions those ids are refering to.